quarkchain/core.py

```diff
@@ -1026,14 +1024,14 @@ def __init__(
 
     def finalize(
         self,
-        coinbase_amount_map=TokenBalanceMap(dict()),
-        coinbase_address=Address.create_empty_account()
+        coinbase_amount_map=dict(),
```

Comment:
this (or even `TokenBalanceMap(dict())`) is bad because default param can be modified and affecting all subsequent calls